docs/getting-started/golden-path.md

```diff
@@ -15,18 +21,6 @@ who are interested.
 Each document in the path will lead to the next so have a look at the bottom of the page
 for the 'Continue to ...' link to keep going.
 
-Continue to download and install...  
-***WINDOWS***
-
-* [Paymetheus](/getting-started/install-guide#windows-installer)
-* [Command Line](/getting-started/install-guide)
-
-***MAC***
-
-* [Decrediton](https://github.com/decred/decrediton)
-* [Command Line](/getting-started/install-guide#)
-
-***LINUX***
+---
```

Comment:
I think we want the golden path to point out the os specific things here since it is supposed to give people the fastest path through thing.

I think if you want to change that bit I'd like other people to say that they want it as well.